webview/src/experiments/components/App.test.tsx

```diff
@@ -1081,9 +1086,9 @@ describe('App', () => {
       renderTableWithoutRunningExperiments()
 
       clickRowCheckbox('4fb124a')
-      clickRowCheckbox('1ba7bcd', true)
```

Comment:
[F] This label no longer exists, the label is now "workspace" as per experiments running in the workspace.

Reply:
[I] We should wrap these somewhere so they are easier to change if we ever have to do it again.